standup/api/admin.py

```diff
@@ -5,4 +5,8 @@
 
 @admin.register(SystemToken)
 class SystemTokenAdmin(admin.ModelAdmin):
-    pass
+    list_display = ('token', 'summary', 'enabled', 'contact', 'created')
+    list_editable = ('enabled',)
+    list_filter = ('enabled',)
+    readonly_fields = ('token',)
+    ordering = ('created',)
```

Comment:
Yeah. I just did a quick initial fleshing-out pass. I'm sure we'll want more tweaks, but I also wanted to see something nicer than default in my local testing :)